aiozk/test/test_session.py

```diff
@@ -1,33 +1,107 @@
 import asyncio
 from unittest import mock
 
-import aiozk
+import aiozk.session
 
 import pytest
+import asynctest
```

Comment:
Please fix tests by adding new package to Dockerfile

Reply:
ouch, would you mind if I add one more test dependency?
cause it's much more convenient to mock objects using `asynctest` instead of trying to `monkeypatch` existing.

Reply:
Sure, no problem. at all.

Personally, I'm more functional/integration tests type of person. In tests like NoNode exception, it is so easy to me just test it on real zk than trying to figure out how and what to mock. 

So your approach shows me more ways to test things :smiley: